dev/helpers/mssql/ReadRequiredMigrations.sql

```diff
@@ -0,0 +1,22 @@
+CREATE OR ALTER PROCEDURE ReadRequiredMigrations
+    @MigrationsFile NVARCHAR(MAX)
+AS
+BEGIN
+    SET NOCOUNT ON;
+
+    CREATE TABLE #InputMigrations (Filename NVARCHAR(MAX))
+
+    -- This is only run locally in dev environments so we are not concerned with SQL injection attacks here.
+    -- This should not be reused in any other context.
+    DECLARE @bulkInsertSql NVARCHAR(4000) = 'BULK INSERT #InputMigrations FROM ''' + @MigrationsFile + ''' WITH (FIELDTERMINATOR = '';'', ROWTERMINATOR = '';'')';
+    EXEC(@bulkInsertSql);
```

Comment:
@withinfocus @rkac-bw is this reasonable or should we just escape it and avoid the explanation?

Reply:
I think it's fine.